Bigtable/src/Connection/ConnectionInterface.php

```diff
@@ -145,7 +150,12 @@ public function dropRowRange(array $args);
     /**
      * @param array $args
      */
-    public function waitForReplication(array $args);
+    public function checkConsistency(array $args);
```

Comment:
As I said, this is already checked in. You can omit this. Same for generateConsistencyToken.
You'd better sync your workspace with the current bigtable branch.